boards/arm/sama5/giant-board/configs/sdmmcnsh/defconfig

```diff
@@ -0,0 +1,143 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_AUDIO_FORMAT_MP3 is not set
+# CONFIG_DISABLE_OS_API is not set
+# CONFIG_NSH_ARGCAT is not set
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+# CONFIG_NSH_CMDPARMS is not set
+# CONFIG_SAMA5_UART0 is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="giant-board"
+CONFIG_ARCH_BOARD_GIANT_BOARD=y
+CONFIG_ARCH_BUTTONS=y
```

Comment:
Refresh configuration.  There should be not button settings.

Did you test this configuration.  If not, please remove it.

Reply:
@patacongo will fix